ParisBNF/et/BNFet343.xml

```diff
@@ -25,9 +25,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <collection>Manuscrits orientaux</collection>
                         <collection>Fonds éthiopien</collection>
                         <collection>Griaule</collection>
-                        <idno>BnF Éthiopien 343 (Griaule 39)</idno>
+                        <idno>BnF Éthiopien 350 (Griaule 46)</idno>
```

Comment:
is this correct? the file is still called 343
if the description is for 350 the filename and xml:id should also be changed (?)